pkgs/by-name/je/jellyfin-ffmpeg/package.nix

```diff
@@ -8,14 +8,79 @@ let
   version = "7.0.2-5";
 in
 
-(ffmpeg_7-full.override {
+(ffmpeg.override {
   inherit version; # Important! This sets the ABI.
   source = fetchFromGitHub {
     owner = "jellyfin";
     repo = "jellyfin-ffmpeg";
     rev = "v${version}";
     hash = "sha256-cqyXQNx65eLEumOoSCucNpAqShMhiPqzsKc/GjKKQOA=";
   };
+
+  ffmpegVariant = "barebones";
+
+  # https://github.com/NixOS/nixpkgs/pull/354936#issuecomment-2466628369
+  # https://github.com/jellyfin/jellyfin-ffmpeg/blob/jellyfin/debian/rules
+  # --enable-gmp # not supported by our ffmpeg drv
+  withDoc = false;
```

Comment:
I find the list of things here quite excessive. Maybe we use small and enable the 5 extra things we need?

Reply:
Headless and small have many options that are not enabled in this.

Reply:
The intention is to mirror the upstream config. I'll add a comment explaining this.